examples/basic-globalize-compiler/README.md

```diff
@@ -0,0 +1,26 @@
+# Basic Globalize Compiler example
+
+On this example, we assume you know the Globalize usage basics. So, it focuses on
+the Globalize Compiler and on the Globalize runtime modules.
```

Comment:
Yeap, all the documentation throughout this repo has been using hard wrapping similarly to JS styles ("Lines should be no longer than 80 characters, and must not exceed 100 (counting tabs as 4 spaces). There are 2 exceptions, both allowing the line to exceed 100 characters").

Do we have a rule for that somewhere? I'm just curious.